net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/runtime/preferences/impl/PreferencesManagerImpl.java

```diff
@@ -665,7 +668,9 @@ private void storeRuleSetInStateLocation(RuleSet ruleSet) {
             IPath ruleSetLocation = plugin.getStateLocation().append(PREFERENCE_RULESET_FILE);
             out = new FileOutputStream(ruleSetLocation.toOSString());
             IRuleSetWriter writer = plugin.getRuleSetWriter();
-            writer.write(out, ruleSet);
+            RuleSets ruleSets = new RuleSets();
```

Comment:
Not sure, if we need this now, since this is the ruleset from the "Rule Configuration"/Global Rule Management... There we still only maintain exactly one ruleset...

Reply:
Could be. I am not 100% sure when it comes to the preferences. I just didn't want to break the code :) I can remove it if you think that is best.